grails-app/assets/javascripts/filters-container.js

```diff
@@ -21,5 +21,12 @@ var filtersContainer = {};
 
     this.init = function() {
         this.container = document.getElementById('filters-container');
+
+        if(this.container === null) {
+            // Something went wrong, pack your bags.
```

Comment:
The way bie-hub is composed leaves opportunity for filters container not existing, but js getting loaded in such order that can break this.

Reply:
And that is sad